README.md

```diff
@@ -1,6 +1,6 @@
 ![logo](https://static.creatordev.io/logo-md-s.svg)
 
-----------
+[![Build Status](http://jenkins.creatordev.io/buildStatus/icon?job=CreatorDev/w2-remoterelay/master)](http://jenkins.creatordev.io/job/CreatorDev/job/w2-remoterelay/job/master/)
```

Comment:
This set of changes are all technically wrong. The badge is per branch showing the build status of each branch as you switch in the ui. On the dev branch it should point to dev and on the master branc hpoint to master.